common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/Utils.java

```diff
@@ -42,4 +47,27 @@ public static String sanitizeBoolean(String value, DataField.DataTypeEnum dataTy
     }
     return value;
   }
+
+  /**
+   * Check if prediction interval is enabled or not.
+   */
+  public static boolean isPredictionIntervalEnabled() {
+    return isPredictionIntervalEnabledFromEnv() || isPredictionIntervalEnabledFromProperty();
+  }
+
+  private static boolean isPredictionIntervalEnabledFromEnv() {
+    try {
+      return Boolean.getBoolean(PREDICTION_INTERVAL_ENABLE_PROPERTY);
+    } catch (Exception ignored) {
+      return false;
+    }
+  }
+
+  private static boolean isPredictionIntervalEnabledFromProperty() {
+    try {
+      return Boolean.parseBoolean(System.getenv(PREDICTION_INTERVAL_ENABLE_ENV_VAR));
+    } catch (Exception ignored) {
+      return false;
+    }
```

Comment:
Similar to my last comment above, please avoid consuming arbitrary exceptions.  In this case, you can get by without any try-catch, propagating runtime exceptions (like `SecurityException`) if and when they come up.

Reply:
make sense